context.go

```diff
@@ -289,6 +289,69 @@ func (c *Context) GetStringMapStringSlice(key string) (smss map[string][]string)
 	return
 }
 
+// Force Type convert
+func StringToInt(s string) (i int) {
```

Comment:
This could be:
```go
func StringToInt(s string) (i int) {
	i, _ = strconv.Atoi(s)
	return
}
```
or even better:

```go
func StringToInt(s string) int {
	i, _ = strconv.Atoi(s)
	return i
}
```
as you shouldn't use named returns...